src/modality-agnostic-files/provenance.md

```diff
@@ -0,0 +1,485 @@
+    - [Provenance records for fMRI preprocessing using `fMRIPrep`](https://github.com/bclenet/bids-examples/tree/BEP028_fmriprep/provenance_fmriprep) - [Associated Pull Request#502](https://github.com/bids-standard/bids-examples/pull/502)
+
+    Further datasets are available from
+    the [BIDS examples repository](https://bids-website.readthedocs.io/en/latest/datasets/examples.html#provenance).
+
+## Overview
+
+### Goals
+
+This part of the BIDS specification is aimed at describing the provenance of a BIDS dataset. This description is retrospective: it describes a set of steps that were executed in order to obtain the dataset (this is different from prospective descriptions of workflows that could for instance list all sets of steps that can be run on this dataset).
+
+### Principles for encoding provenance in BIDS
+
+-   Provenance information SHOULD be included in a BIDS dataset when possible.
+-   If provenance records are included, these MUST be described using the conventions detailed by this specification.
+-   Provenance records MAY be used to reflect the provenance of a full dataset and/or of specific files at any level of the BIDS hierarchy.
+-   Provenance information SHOULD be de-identified as necessary.
+
+### Definitions
+
+**Provenance record**: provenance metadata consists in sets of records. These provenance records can be of 4 types:
+
+-   `Activity`: Activities represent the transformations that have been applied to the data.
+-   `Entity`: Entities represent input or output data for Activities.
+-   `Software`: The Software record describes a software package.
+-   `Environment`: Environments specify the software environment in which the provenance record was obtained.
+
+Provenance records are described as JSON objects in BIDS. This is detailed in the [Provenance files section](#provenance-files).
+
+!!! example
+    The following graph represents examples of links between provenance records. In this example, `Entities` *sub-001_brainmask.nii* and *sub-001_T1w.nii* represent files used by `Activity` *Brain extraction* to generate another file represented by `Entity` *sub-001_T1w_preproc.nii*. `Activities` *Brain extraction* and *Move to MNI* were associated with the `Software` *FSL* and used `Environment` *Linux* as a software environment.
+    ![](../images/provenance_definitions_graph.png)
+
+**Provenance group**: refers to a collection of provenance records corresponding to a consistent set of processings. Defining multiple provenance groups is appropriate when separate sets of processings have been performed on data, and that they differ in purpose or chronology.
+
+## Provenance in sidecar JSON files
+
+Provenance metadata CAN be stored inside the sidecar JSON of any BIDS file (or BIDS-Derivatives file) it applies to.
+This metadata only describes the provenance of the BIDS file.
+
+Any sidecar JSON file CAN include the following keys:
```

Comment:
```suggestion
Any sidecar JSON file MAY include the following keys:
```

per https://bids-specification--2099.org.readthedocs.build/en/2099/common-principles.html#definitions where no `CAN` is defined . Change here and elsewhere in the text